src/tools/OptionsSchema.json

```diff
@@ -176,35 +176,26 @@
     },
     "LaunchBrowserPlatformOptions": {
       "type": "object",
+      "required": [ "command" ],
       "properties": {
         "command": {
           "type": "string",
-          "description": "The command to execute for launching the web browser",
+          "description": "The executable to launching which will start the web browser",
```

Comment:
This sounds weird. Did you want `The executable which will start the web server`?

Reply:
Sounds reasonable.